### PR TITLE
NIAD-1085: release 0.0.1 into main

### PR DESCRIPTION
**Pull request by bartek-sarul**
## Description

Please explain the changes you have made here.

## Jira Ticket

Please add a link to the related Jira ticket (if any).

## Checklist

These are items (excluding GitHub Checks) which should be confirmed before a branch is ready to merge.

- [ ] Acceptance Criteria met
- [ ] Commit messages are meaningful
- [ ] Manually tested
- [ ] Self-reviewed your code
- [ ] Code reviewed from two other developers
- [ ] If your pull request depends on any other, please link them in the description
- [ ] main branch is passing/green on CI
- [ ] Add/update any relevant documentation

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [0.0.1] - 2021-02-22
+### Added
+- First release of Lab Results Adaptor
+- NHSACK is not produced
```
Comment (andrewwa-kainos) [inline CHANGELOG.md:12]: Not sure about "Added: this thing is not added"
Maybe list that NHSACK is not produced as a known issue?

File: mesh/README.md [hunk 2]
```diff
@@ -16,6 +16,18 @@ accordingly)
 For all commands `[mailbox_id]` is the id of the mailbox used for the request. The `MAILBOX_PASSWORD` environment
 variable must be correct for the given id.
 
+### Environment set up
+
+Invoke the desired `env.sh` file to load environment variables that will be used by the mesh client
+
+You own values:
```
Comment (matthewdoreilly) [inline mesh/README.md:23]: Should this be "Your own values"?

File: mesh/README.md [hunk 3]
```diff
@@ -16,6 +16,18 @@ accordingly)
...
+
+    source env.sh
```
Comment (matthewdoreilly) [inline mesh/README.md:25]:
```suggestion
    `source env.sh`
```

File: mesh/README.md [hunk 4]
```diff
@@ -16,6 +16,18 @@ accordingly)
...
+
+Predefined fake mesh values:
+
+    source env.fake-mesh.sh
```
Comment (matthewdoreilly) [inline mesh/README.md:29]:
```suggestion
    `source env.fake-mesh.sh`
```

File: release/DOCKERHUB.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+## Clone the repository
+
+```bash
+git clone https://github.com/nhsconnect/integration-adaptor-lab-results.git
+```
+
+## Pull the latest changes and checkout the release tag
+
+Every tagged container on Docker hub has a corresponding tag in the git repository. Checkout the tag of the release
```
Comment (matthewdoreilly) [inline release/DOCKERHUB.md:26]:
```suggestion
Every tagged container on Docker Hub has a corresponding tag in the GitHub repository. Checkout the tag of the release 
```

File: release/DOCKERHUB.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+you are testing to ensure compatibility with configurations and scripts.
+
+```bash
+git pull
+git checkout 0.0.1
+```
+
+## Configure the application
+
+docker-compose.yml is preconfigured with environment values to run a standalone version of the adadptor using fake mesh.
```
Comment (matthewdoreilly) [inline release/DOCKERHUB.md:36]:
```suggestion
`docker-compose.yml` is preconfigured with environment values to run a standalone version of the adaptor using fake mesh.
```

File: release/DOCKERHUB.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+If you would like to run the adaptor against any other Mesh service or using any other database or message queue broker, modify required environment variables as described in to  [README](https://github.com/nhsconnect/integration-adaptor-lab-results/blob/0.0.1/README.md)
```
Comment (matthewdoreilly) [inline release/DOCKERHUB.md:38]:
```suggestion
If you would like to run the adaptor against any other Mesh service or using any other database or message queue broker, modify required environment variables as described in the [README](https://github.com/nhsconnect/integration-adaptor-lab-results/blob/main/README.md)
```

The linked README should be to the one in the main branch, not the release branch?

File: release/DOCKERHUB.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+```bash
+./logs.sh
+```
+
+## Run the tests
+
+We provide shell scripts in the release/tests directory to help you start testing.
+
+* `healthcheck.sh` verifies that the adaptor's healthcheck endpoint is available
+* `send_message.sh` puts an example Edifact message on the fake mesh allowing it to be consumed and translated to FHIR by the adaptor
```
Comment (matthewdoreilly) [inline release/DOCKERHUB.md:66]:
```suggestion
* `send_message.sh` puts an example EDIFACT message on the fake mesh allowing it to be consumed and translated to FHIR by the adaptor
```

File: mesh/README.md (hunk 2 above)
Comment (graham-andrew) [inline mesh/README.md:23]: Should this be "your own values"?

**Review by matthewdoreilly — COMMENTED** [review]
Only minor changes

File: CHANGELOG.md (hunk 1 above)
Comment (bartek-sarul) [reply CHANGELOG.md:12]: It has already been released. We'll remove that with v0.2

**Comment by bartek-sarul** [comment]
Closing as branch name is wrong. Reopening with applied code review fixes in https://github.com/nhsconnect/integration-adaptor-lab-results/pull/66

**Pull request closed without merging**